src/core_tests.rs

```diff
@@ -14,6 +14,7 @@
 //
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
+#![cfg(test)]
```

Comment:
Makes sense - how do you do such double `cfg`s, though? Just two `cfg` attrs one below the other?